custom_components/mikrotik_router/coordinator.py

```diff
@@ -284,13 +284,16 @@ def __init__(self, hass: HomeAssistant, config_entry: ConfigEntry):
         self.host_tracking_initialized = False
 
         self.support_capsman = False
+        self.support_wifi = False
         self.support_wireless = False
         self.support_wifiwave2 = False
         self.support_ppp = False
         self.support_ups = False
         self.support_gps = False
+        self._wifimodule = None
```

Comment:
this would break all current installations. default path needs to be wireless
`self._wifimodule = "wireless"`

Reply:
Fixing